src/main/resources/edu.umd.lib.wufooconnectorcamel.cfg

```diff
@@ -10,13 +10,12 @@ wufoo.handshake_key=testing
 
 # Wufoo Sysaid Field Mapping 
 # Format for wufoo.sysaid.mapping is : formhash=filepath,form2hash=filepath2
-wufoo.sysaid.mapping=
+wufoo.sysaid.mapping=q1li2eu01gm1gnx=/apps/git/wufoo-camel-connector/src/main/resources/Wufoo-Sysaid-Mapping.properties,q1li2eu01gm1gny=Wufoo-Sysaid-Mapping.properties
 
-#Properties required for SysAid Connection 
-sysaid.servicename=wufoo-listener
-sysaid.url=https://libticketingdev.umd.edu/api/v1/
-sysaid.username=
-sysaid.password=
+#Properties required for SysAid Form Submit
+sysaid.webformurl=https://libticketingdev.umd.edu/webformsubmit?pageEncoding=utf-8
+sysaid.accountid=
+sysaid.formid=8cb90ec:1565644c31d:-7ffb
```

Comment:
Leave sysaid.webformurl and sysaid.formid blank. Add documentation on how to find these values from the sysaid web interface.